docs/UI/GetCapabilities/index.md

```diff
@@ -60,7 +60,7 @@ GetCapabilities
     "displayCapabilities" :
     {
       "displayType" : "GEN2_8_DMA",
-      "textFields" : ["mainField1", "mainField2", "mediaclock", "mediaTrack", "alertText1", "alertText2", "alertText3", "scrollableMessageBody", "initialInteractionText", "navigationText1", "navigationText2", "audioPassThruDisplayText1", "audioPassThruDisplayText2", "notificationText"],
```

Comment:
Hmmm, seems these are incorrect, there is a full struct for each of these elements:
```
    <struct name="TextField" since="1.0">
        <param name="name" type="TextFieldName" mandatory="true">
            <description>The name that identifies the field. See TextFieldName.</description>
        </param>
        <param name="characterSet" type="CharacterSet" mandatory="true">
            <description>
                The set of characters that are supported by this text field.
                All text is sent in UTF-8 format, but not all systems may support all of the characters expressed by UTF-8.
                All systems will support at least ASCII, but they may support more, either the LATIN-1 character set, or the full UTF-8 character set.
            </description>
        </param>
        <param name="width" type="Integer" minvalue="1" maxvalue="500" mandatory="true">
            <description>The number of characters in one row of this field.</description>
        </param>
        <param name="rows" type="Integer" minvalue="1" maxvalue="8" mandatory="true">
            <description>The number of rows of this field.</description>
        </param>
    </struct>
```

Reply:
good catch, i added real output in https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/237/commits/1c79899bee6da094ffc52e21c866aa55f82a72fc